backend/app/UseCases/Project/GetProjectUseCase.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UseCases\Project;
+
+use App\Models\Project;
+
+final class GetProjectUseCase
+{
+    public function invoke(
+        int $projectId
+    ): Project {
+        $project = Project::find($projectId);
+
+        return $project;
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method App\UseCases\Project\GetProjectUseCase::invoke() should return App\Models\Project but returns App\Models\Project|null.